### ISSUE TITLE
Moving Diffusionbee with .diffusionbee folder to an external drive

### ISSUE THREAD

**Issue by tuncyalcin** [divamgupta/diffusionbee-stable-diffusion-ui#480]
Hi, is it possible to move the entire app with extras to an external drive? I'm running out of storage with my m1 macbook and i really want to use civitai models to generate. Can someone please help? Thanks

**Comment by nightwingrox711** [comment]
yes, but its kind of wierd, took me a while to figure out how to do it a couple of months ago

first your gonna wanna go to your local folder and press command+shift+.
'.'
to be able to view dotfiles, from there youll be able to see your .diffusionbee dotfile
move that to your external drive, along with the diffusion bee app

and then go to your local folder via terminal, or open a terminal at your local folder in finder, and type:

ln -s /Volumes/(name of your external drive/.diffusionbee ~/.diffusionbee

diffusionbee will try to make a new dotfile because it doesn't believe it has one, because you removed it from the local folder, so this command should just forward it to the new location of the dotfiles

**Comment by tuncyalcin** [comment]
thank you, i'll try.

**Comment by kappesante** [comment]
omg thanks
